front/src/applications/operationalStudies/v2/components/ScenarioHeader.tsx

```diff
@@ -15,15 +15,30 @@ import useAuth from 'utils/hooks/useAuth';
 type ScenarioHeaderProps = {
   scenario: ScenarioResponse;
   infra: InfraWithState;
+  toggleTimeTable: () => void;
+  toggleConflictsList: () => void;
 };
 
-const ScenarioHeader = ({ scenario, infra }: ScenarioHeaderProps) => {
+const ScenarioHeader = ({
+  scenario,
+  infra,
+  toggleConflictsList,
+  toggleTimeTable,
+}: ScenarioHeaderProps) => {
   const { username } = useAuth();
   const { openModal } = useModal();
   const navigate = useNavigate();
 
   const { t } = useTranslation('operational-studies');
-  const [activeBoards, setActiveBoards] = useState<string[]>([]);
+  const [activeBoards, setActiveBoards] = useState<string[]>([
```

Comment:
Would be nicer to have a union instead of plain strings here, but this is out-of-scope since this is existing code from before this PR.

```ts
type Board = 'trains' | 'map' | …;
```